src/main/java/uk/gov/hmcts/ccd/endpoint/exceptions/RestExceptionHandler.java

```diff
@@ -42,11 +43,23 @@ public RestExceptionHandler(AppInsights appInsights) {
     @ExceptionHandler(Exception.class)
     @ResponseBody
     public ResponseEntity<HttpError> handleException(final HttpServletRequest request, final Exception exception) {
-        LOG.error(exception.getMessage(), exception);
+        LOG.error("data store request processing exception", exception);
         appInsights.trackException(exception);
```

Comment:
What should be the level of LOG L32 said LOG.warn and here it is LOG.error

Reply:
we agreed to change this with Valentin. The thing is that handException handles unknown exceptions which for sure were not logged anywhere else in the code. 
handleApiException instead handles exceptions that probably have already been logged. So we changed to warn. 
But I've noticed lots of duplicated error logging, so we really should revisit our strategy